go/vumitools/contact/models.py

```diff
@@ -322,12 +329,12 @@ def new_contact_for_addr(self, delivery_class, addr):
         return self.new_contact(**field_dict)
 
     @Manager.calls_manager
-    def contact_for_addr(self, delivery_class, addr, create=True):
+    def contact_for_addr_field(self, field, value, create=True):
         """
-        Returns a contact from a delivery class and address, raising a
+        Returns a contact from a field (address type) and address, raising a
         ContactNotFoundError exception if the contact does not exist.
         """
-        field, value = contact_field_for_addr(delivery_class, addr)
+        field, value = normalize_addr(field, value)
```

Comment:
I think I would like ``contact_for_addr_field`` to not call ``normalize_addr`` because it's meant to be a low-level utility method. I'm happy for particular higher-level things to call ``normalize_addr`` if they like.